plugins/bundle/plugin.go

```diff
@@ -540,6 +540,8 @@ func (p *Plugin) process(ctx context.Context, name string, u download.Update) {
 	if etag, ok := p.etags[name]; ok && u.ETag == etag {
 		p.log(name).Debug("Bundle load skipped, server replied with not modified.")
 		p.status[name].SetError(nil)
+		// proposed fix to check plugin readiness on 304
+		// p.checkPluginReadiness()
```

Comment:
Alternatively you could fix this in `Reconfigure` as well I think but this looks fine.